v_0_1_0/master_template.go

```diff
@@ -1129,16 +1129,16 @@ coreos:
       ExecStartPre=-/usr/bin/docker rm  $NAME
       ExecStartPre=-/usr/bin/docker pull $IMAGE
       ExecStart=/usr/bin/docker run \
-        -v /etc/giantswarm/g8s/ssl/etcd/:/etc/etcd \
+        -v /etc/kubernetes/ssl/etcd/:/etc/etcd \
```

Comment:
Has this implications for the flannel-operator and the pods started for flanneld?

Reply:
no, as this is the guest cluster only

we still have inconsistency  of this folder (red is bare-metal host cluster and green is guest cluster)

nothing really changed i just badly copied